Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/sources/fallback/DefaultAppConfigSourceTest.kt

```diff
@@ -82,7 +82,8 @@ class DefaultAppConfigSourceTest : BaseIOTest() {
 
     @Test
     fun `exceptions when getting the default config are rethrown`() = runBlockingTest {
-        every { assetManager.open("default_app_config_android.bin") } throws FileNotFoundException("default_app_config_android.bin does not exist")
+        every { assetManager.open("default_app_config_android.bin") } throws
+            FileNotFoundException("default_app_config_android.bin does not exist")
```

Comment:
Changes regarding formatting in Test classes are due to `detekt` newer version which now check Test classes too